https://github.com/keithamus/parse-packagejson-name/pull/15

diff --git a/index.d.ts b/index.d.ts
index e8dcfb3cd3c3b7709fec4ce59192ef0fa8410857..c4c1cefde747306dabe6a2008cdb95a0641c1ea5 100644
--- a/index.d.ts
+++ b/index.d.ts	
@@ -6,7 +6,7 @@ export interface ParsedPackageName {
 }
 
 declare function parsePackageJsonName(
-  name: string | { name?: string }
+  name?: string | { name?: string }
 ): ParsedPackageName;
 
 export = parsePackageJsonName;
